prog/vnet/subnet_nexus.rb

```diff
@@ -20,6 +20,14 @@ def self.assemble(project_id, name: nil, location: "hetzner-hel1", ipv6_range: n
     DB.transaction do
       ps = PrivateSubnet.create(name: name, location: location, net6: ipv6_range, net4: ipv4_range, state: "waiting") { _1.id = ubid.to_uuid }
       ps.associate_with_project(project)
+      FirewallRule.create_with_id(
```

Comment:
I thought we want to set this for all existing subnets, not for new ones. Shouldn't the new VMs be non-accessible from public internet by default as well.

Reply:
Nope, because we're not implementing full firewall functionality, yet. They won't have a way to open it up to access for other ports than SSH, if we do it like that :)